src/shared/components/palette/history/event-log/event-log-details.component.html

```diff
@@ -5,7 +5,7 @@
       <th><span class="text-16">Date</span></th>
       <td>
         <div class="tooltip text-16">{{event.timestamp | ccdDate : 'local'}}
-          <span class="tooltiptext text-16">Local: {{event.timestamp | ccdDate : 'local'}}</span>
+          <td><span class="text-16">{{event.user_first_name | titlecase}} {{event.user_last_name | uppercase}}</span></td>
```

Comment:
A question, this div is for timestamp, but the behavior is now the sub item is changed to displaying firstName lastName, is that expected behavior? Given that this is a generic component, need to estimate the side effects.